helm/istio/charts/tracing/templates/deployment-jaeger.yaml

```diff
@@ -113,6 +113,43 @@ spec:
 {{- end }}
       serviceAccountName: jaeger
       containers:
+      # OAuth proxy
```

Comment:
As we're moving to the Jaeger operator, does the operator support this configuration?